### ISSUE TITLE
Cannot check archive dir free space

### ISSUE THREAD

**Issue by kukat** [ericaltendorf/plotman#153]
plotman dirs could not get the archive dir information 

$ plotman dirs
```
    tmp       ready   phases
/mnt/ssd/00      --   1:3   
/mnt/ssd/01      --   3:6   
/mnt/ssd/02      --   3:3   
/mnt/ssd/03      --   2:5   
/mnt/ssd/04      --   2:0   
    dst       plots   GBfree      inbnd phases       pri
/mnt/dst/00   1       781      1:3 2:0 2:5 3:3 3:6   38 
archive dirs free space:
```

my archive configurations in plotman.yaml

```
        archive:
                rsyncd_module: T12A
                rsyncd_path: /mnt/T12A
                rsyncd_bwlimit: 100000  # Bandwidth limit in KB/s
                rsyncd_host: chia-farm
                rsyncd_user: chia
```

I'm not sure why the `grep` doesn't work for me in [archive.py](https://github.com/ericaltendorf/plotman/blob/main/src/plotman/archive.py#L52)
`ssh chia@chia-farm df -aBK | grep " /mnt/T12A/"` doesn't return any ouput

I can verify the `rsyncd_path` was existed on the remote machine by running the command without grep `ssh chia@chia-farm df -aBK`

```
Filesystem        1K-blocks        Used    Available Use% Mounted o
...
tmpfs              1605960K          4K     1605956K   1% /run/user/1000
/dev/sda       11716272064K 2551022960K  9165232720K  22% /mnt/T12A
/dev/sdb       11716272064K 1488082892K 10228172788K  13% /mnt/T12B
binfmt_misc              0K          0K           0K    - /proc/sys/fs/binfmt_misc
```

I did a few tests, and here is what works for me:
1. check the `rsyncd_path` directly: `ssh chia@chia-farm df -aBK /mnt/T12A`
2. remove the trailing slash: `ssh chia@chia-farm df -aBK | grep " /mnt/T12A"`

just want to confirm is this just me and my setup? 
or a bug in the code? if so I'm happy to create a PR to address the issue.

**Comment by altendky** [comment]
`rsyncd_path` is expected to be given a directory which contains mount points.  Not a single mount point directly.  This allows plotman to identify multiple drives and choose which to archive to based on available space and optionally the index configuration value.  Given the mounts I see here I think I would expect you to have `rsyncd_path: /mnt` such that plotman would find `/mnt/T12A` and `/mnt/T12B` as archival target candidates.  Of course, `/mnt` may well include other drives you don't want used for archival.  When I started using archiving I had to reorganize my mount points so there was a directory containing only archival mount points and nothing else.

Let me know if this gets you going again or not.
